app.rb

```diff
@@ -0,0 +1,131 @@
+require_relative 'rental'
+require_relative 'classroom'
+require_relative 'teacher'
+require_relative 'person'
+require_relative 'book'
+require_relative 'student'
+
+class App
+  def initialize
+    @books = []
+    @persons = []
+    @rentals = []
+  end
+
+  def start_console
+    puts 'welcome to our school Library App!'
+    until list_of_options
+      input = gets.chomp
+      if input == '7'
+        puts 'Thank You for using our school Library!'
+        break
+      end
+
+      option input
+    end
+  end
+
+  def list_all_books
+    puts 'Database is empty! Add a book.' if @books.empty?
+    @books.each { |book| puts "[Book] Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def list_all_persons
+    puts 'Database is empty! Add a person.' if @persons.empty?
+    @persons.each do |person|
+      puts "[#{person.class.name}] Name: #{person.name}, Age: #{person.age}, id: #{person.id}"
+    end
+  end
+
+  def create_person
+    print 'press 1 to create a student, press 2 to create a teacher : '
+    option = gets.chomp
+
+    case option
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'Invalid input. Try again'
+    end
+  end
+
+  def create_student
+    puts 'create a new student'
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp.downcase
+    case parent_permission
+    when 'n'
```

Comment:
- [ ] When creating a student, the app crashes whenever I enter a value for the Has parent permission. Kindly work on that so a value for the parent permission question is entered and a student is saved successfully. 👍🏽

<img width="1212" alt="Screenshot 2023-06-22 at 4 53 56 PM" src="https://github.com/Roland-Ntwali/School-library-app/assets/50941074/addc1a41-aeb0-4641-a076-51931f1d63c5">